scripts/jenkins/ardana/ansible/run-crowbar-tests.yml

```diff
@@ -6,6 +6,10 @@
   gather_facts: True
   vars:
     task: "tempest"
+    subunit_tools_venv: "/var/lib/ardana/subunit-tools-venv"
```

Comment:
Just use something in the `/root` folder. We can worry about unification with Ardana some other time.